mantle/platform/api/packet/api.go

```diff
@@ -333,98 +295,62 @@ After=dev-sda.device
 
 [Service]
 Type=oneshot
-# Prevent coreos-install from validating cloud-config
-Environment=PATH=/root/bin:/usr/sbin:/usr/bin
 
-ExecStart=/usr/bin/curl -fo image.bin.bz2 "%v"
-# We don't verify signatures because the iPXE script isn't verified either
-# (and, in fact, is transferred over HTTP)
+# We don't verify signatures because this might be a random dev image.
```

Comment:
OK for now but I think we should make this an option like it is for `testiso`, then we can default to secure for `--fcos-stream` spawns?

Reply:
There's no mechanism for verifying the iPXE script or the PXE image we use to run the installer.  If you don't trust the web server, the image signature won't help.